### ISSUE TITLE
"Local executor requires the use of POSIX compatible file system" with latest Nextflow

### ISSUE THREAD

**Issue by pinin4fjords** [nf-core/fetchngs#143]
### Description of the bug

Running with nf 23.04.0 (in Tower) produces errors like:

```
Uploading local `bin` scripts folder to s3://nf-tower-workdir/fetchngs/YOUR_ID/tmp/bd/205133e4da5bfe13eb99139d65b8e1/bin
Local executor requires the use of POSIX compatible file system — offending work directory path: s3://nf-tower-workdir/fetchngs/YOUR_ID
```

... before any tasks start.

Pinning Nextflow back like:

```
export NXF_VER=22.10.7
```

... allows the workflow to run. 

### System information

Nextflow Tower Cloud, AWS Batch compute environment

**Comment by pinin4fjords** [comment]
Thanks @drpatelh. We're getting it on rnaseq as well- is there already an issue open on Nextflow somewhere to track resolution? I can't find one right now.

**Comment by drpatelh** [comment]
Yep, it will affect any pipelines that are using a process with `executor 'local'`. I have fixed this in nf-core/rnaseq v3.11.1 already by removing these processes. We need to do the same for this pipeline.

We are tracking the issue internally at the moment but will keep you updated when we push a fix.

**Comment by drpatelh** [comment]
This should be fixed now in [NF v23.04.1](https://github.com/nextflow-io/nextflow/releases/tag/v23.04.1) via https://github.com/nextflow-io/nextflow/commit/6ea1f68c10608e0f9126c83cfaf5c9748bfbb419
